post-processor/vagrant/libvirt_test.go

```diff
@@ -67,3 +72,70 @@ func Test_sizeInMegabytes_WithExabytesUnit(t *testing.T) {
 	assertSizeInMegabytes(t, "1234E", 1234*1024*1024*1024*1024)
 	assertSizeInMegabytes(t, "1E", 1*1024*1024*1024*1024)
 }
+
+func Test_ManyFilesInArtifact(t *testing.T) {
+	p := new(LibVirtProvider)
+	ui := testUi()
+	type testCases struct {
+		Files         []string
+		Format        string
+		FilesExpected []string
+	}
+	testcases := []testCases{
+		{
+			[]string{},
+			"qcow2",
+			[]string{},
+		},
+		{
+			[]string{"test-0"},
+			"vmdk",
+			[]string{"box_0.img"},
+		},
+		{
+			[]string{"test-0", "/test-1", "test-2"},
+			"qcow2",
+			[]string{"box_0.img", "box_1.img", "box_2.img"},
+		},
+	}
+	for _, tc := range testcases {
+		dir, _ := tmp.Dir("pkr")
+		defer os.RemoveAll(dir)
+
+		actifactFiles := []string{}
+		for _, file := range tc.Files {
+			fullFilePath := fmt.Sprintf("%s/%s", dir, file)
+			actifactFiles = append(actifactFiles, fullFilePath)
+			_, err := os.Create(fullFilePath)
+			if err != nil {
+				t.Fatalf("Can't create %s : %s", fullFilePath, err)
+			}
+		}
+
+		artifact := &packersdk.MockArtifact{
+			FilesValue: actifactFiles,
+			StateValues: map[string]interface{}{
+				"diskType":   tc.Format,
+				"diskSize":   "1234M",
+				"domainType": "kvm",
+			},
+		}
+
+		_, metadata, err := p.Process(ui, artifact, "foo")
+
+		if err != nil {
+			t.Fatalf("should not have error: %s", err)
+		}
+		metaDisks := metadata["disks"].([]map[string]string)
+		assert.Equal(t, len(tc.FilesExpected), len(metaDisks),
```

Comment:
While I don't personally hate the construction, I'm not sure it's a good idea to add a new package for such a small change, this could as easily be a `if`, and wouldn't change a lot in the current code.

```go
if len(tc.FilesExpected) != len(metaDisks) {
    t.Errorf("Expected %d disks, but test returned %d", len(tc.FilesExpected, len(metaDisks))
}
```

Reply:
I will change it